src/clj_http/conn_mgr.clj

```diff
@@ -152,16 +143,12 @@
                    (get-keystore-scheme-registry config)
 
                    :else regular-scheme-registry)]
-    (PoolingClientConnectionManager.
-     registry timeout java.util.concurrent.TimeUnit/SECONDS)))
-
-(def dmcpr ConnPerRouteBean/DEFAULT_MAX_CONNECTIONS_PER_ROUTE)
+    (PoolingHttpClientConnectionManager. registry)))
```

Comment:
It looks like `timeout` is no longer being used here. I've made some tests and it seems like that connections will never be closed by connection manager in this case.